### DISCUSSION TITLE
Who wants to be hired (Dec 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#43786] [Ideas]
# What is this?
This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Resume/CV:
  Website:
  Email:

Good luck with interviewing!

---

If you got hired through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a company looking to post a role, use #43787.

*If you're looking for last month's discussion, see #42310. We've had another [*success story*](https://twitter.com/Nutlope/status/1475929984174895105) of a developer being hired!*

**Comment by agonzalo36r** [comment]
### Front-End Developer

Hi, I'm a 6 years-experience front-end developer (the last 3 years on web development), currently focused on React and Next.js. Very interested on the Jamstack architecture, and looking to gain full-stack skills.

**Location:** Lima, Perú / (GMT-5)
**Remote:** Yes
**Willing to relocate:** No
**Technologies:** React, Next.js, Tailwind CSS, Typescript, Directus, Javascript (ES5, ES6), PostgreSQL, Zustand, Redux, Node.js, Express
**Resume/CV:** https://puertaredonda.com/documents/CV_ENG.pdf
**LinkedIn:** https://www.linkedin.com/feed/
**Website/Portfolio:** https://puertaredonda.com
**Email:** agonzalo36r [at] gmail [dot] com

Thanks for the thread! Look forward to hearing from you soon :)

**Reply by jamestrenda** [reply]
I recommended just putting a summary about who you are/what you do on your homepage. No need to get jiggy with it. Clean, simple, to the point—just like the rest of your site. 🙂

**Comment by ogrodev** [comment]
**Hello recruiters!**

If you are looking for a developer that is outstandingly beautiful, muscled, and tall it's not me, but if you are looking for someone creative, smart and very skilled, here I am :)

Location: Rio de Janeiro - Brazil
Remote: Yes
Willing to relocate: Yes
Technologies: NextJS, React, Typescript, SASS, ES6+, Redux, SEO, Bootstrap, Material UI, Figma/XD/Sketch, Prerendering, GraphQL/Rest, PostgreSQL/MySQL, Strapi, Builder.io, Websockets... The list goes on 🚀 
Resume/CV: [HereIsWhyPedroShouldBeYourNextDev.com](https://inpedromendes.notion.site/CV-Pedro-Mendes-EN-30329a593a974f0aa21bda185979edd6)
Website: [Linkedin (it's not my website unfortunately 😂)](https://linkedin.com/in/inpedromendes)
Email: pedroh.seven@gmail.com

**Comment by jensen** [comment]
Location: Vancouver, BC
Remote: Yes
Willing to relocate: No
Technologies: Full-Stack JavaScript (Check GitHub)
Resume/CV: https://www.linkedin.com/in/karljensendev/
Website: https://github.com/jensen

**I am only looking for contract work at this time.**

**Comment by yinkakun** [comment]
Remote: Yes
Willing to relocate: Yes
Location: Nigeria (GMT+1)

Hi there! My name is Yinka Adedire.

I have experience working with NextJS, GatsbyJS, Headless CMS, JavaScript/TypeScript, React, CSS/SCSS, Tailwind CSS, AWS S3, and Git. 

I am passionate about building beautiful and functional websites that are easy to use and maintain.
I have a proven track record of delivering high-quality projects on time and on budget, and I am always looking for new challenges to tackle.

Email:  yinkakun@gmail.com.
Website: https://dev.to/yinks
Resume: https://www.yinka.codes/01-yinka-adedire-resume.pdf

**Comment by MorganJay** [comment]
Location: Lagos, Nigeria (GMT+1)
Remote: Yes
Willing to relocate: Yes
Technologies: Microsoft .NET (Core) & C#, JavaScript, TypeScript, React.js, Redux (Thunk, Saga, Toolkit), Next.js, Java, Microservices, Mediatr, Microsoft SQL Server, MySQL, MongoDB, Quartz.NET, Hangfire, Swagger, Redis, Git, BitBucket, GCP(Firebase Firestore & Cloud Messaging), GitHub, Bitbucket, JIRA, Postman, Docker, AWS services, Netlify, Vercel, Azure, ... The list goes on evolving🚀
Resume/CV: [My Resume](https://docs.google.com/document/d/1uh4yxOhKv8MYM5J0Fru1XvangbhNikJ3/edit?usp=sharing&ouid=110605575482477397792&rtpof=true&sd=true)
Website: [jamesmorgan.dev](https://www.jamesmorgan.dev)
Email: [jetmorgan.jm@gmail.com](mailto:jetmorgan.jm@gmail.com)

**Comment by brvarner** [comment]
Hello! I'm a full stack developer originally from Birmingham, AL who now lives in Chicago. I'd love to hear from you about any projects that require full-time or contract devs.

Location: Chicago, IL. US
Remote: Yes
Willing to relocate: No
Phone: +1 205 542 6109
Technologies: JavaScript, React, React Native, Next.JS, Three.js, MySQL, Google Firebase, AWS, GraphQL, MUI, Tailwind CSS, Node.js, Express.js
Resume/CV: https://drive.google.com/file/d/1XsdLkNpT92nxmlSO_fIJAKwyJirHR5FJ/view?usp=sharing
LinkedIn: https://www.linkedin.com/in/brandonvarneral/
Website: https://bvarner.dev/
Email: [bvarner4592@gmail.com](mailto:bvarner4592@gmail.com)

**Comment by LucasWongC** [comment]
Location: Toronto, ON, Canada
Remote: Yes
Willing to relocate: No
Technologies: JavaScript/TypeScript, React, Next.js, Node.js, CSS/SCSS, Tailwind CSS, MySQL, PostgreSQL, MongoDB, AWS, Git
Resume/CV: https://drive.google.com/file/d/15SuFIj1LbmPtniNwru3VCgYTE617EQok/view
Website: https://lucaswong-portfolio.netlify.app/
Email: [lucas.wong.1118@gmail.com](mailto:lucas.wong.1118@gmail.com)

**Comment by venus01015** [comment]
Hi, My name is Jin Wang and I am a senior full stack developer with over 8 years of experience in software development using practical frameworks, most recent focusing on MERN stack, 3D visualization and blockchain.

Location: Toronto, Canada
Remote: Yes
Willing to relocate: No
Technologies: React, Next.js, Nodejs, Express.js, Typescript, MongoDB, Tailwind CSS, WebGL, Three.js, Babylon.js, Python
Resume/CV: https://cvdesignr.com/p/618960fde7a23
Website: https://funny-dev-site.vercel.app/
Email: [darkrut22@gmail.com](mailto:darkrut22@gmail.com)

**Comment by zhyd1997** [comment]
Hi 👋 there

I am Yadong Zhang from China and am a frontend developer, also a freelancer.

Currently just looking for a **full-time** job, let me introduce myself more:

After graduated in July 2019 with Bachelor's degree (no CS background), I started my CS journey for a year, and then got my first job at [**Compass**](https://github.com/UrbanCompass): in March 2021, but just working for a few months, that was a good experience in the listed company.

In the second half of the year, of course, I applied many job opportunities (Including ByteDance and RedHat) but unfortunately, all of them failed, then I found a part-time job at [**Stargately**](https://github.com/stargately/ ) (web-boommo app, which is private in GitHub), I learned that every developer should have his/her own product so I launched my own side project called [**SoftMaple**](https://github.com/softmaple/softmaple), which is a paper typesetting editor for professors and students. The project is promoted by Github in Dec 2021: https://github.blog/2022-01-21-release-radar-dec-2021/

And in 2022, I became a freelancer at Web3 field:

Contributed to

- **Superfluid** ecosystem: (bug fixes and fixtures)

  - [Ricochet-Exchange](https://github.com/Ricochet-Exchange/ricochet-frontend) - _Polygon_ Network

  - [Geo-Web](https://github.com/Geo-Web-Project/cadastre) - _Optimism_ Network

- **Synthetix** ecosystem: (bug fixes and features)

  - [Kwenta](https://github.com/Kwenta/kwenta) - _Optimism_ Network

- **Uniswap** (small bug fixes)

  - https://github.com/Uniswap/interface

Also traditional web2 open source project:

- **Storybook** (small bug fixes)

1. [storybook](https://github.com/storybookjs/storybook)

2. [design-system](https://github.com/storybookjs/design-system/)

3. [eslint-plugin-storybook](https://github.com/storybookjs/eslint-plugin-storybook)
4. [mdx2-csf](https://github.com/storybookjs/mdx2-csf)

- **ESLint** ecosystem (small bug fixes)

  - [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint)

Location: **China**
Remote: **Yes**
Willing to relocate: **Yes**
Technologies: **React**, **Nextjs**, **Emotionjs**, **TypeScript**
Resume/CV: https://www.linkedin.com/in/yadong-zhang-48a474154/
Website: https://github.com/zhyd1997
Email: zhyd007@gmail.com

**Comment by swopno1** [comment]
Location: Chittagong, Bangladesh
Remote: Yes
Willing to relocate: No
Technologies: JavaScript, React, TailwindCSS, Node.js, Expressjs, Next.js, Netlify, Vercel, MongoDB, Web3, Google Apps Script
Resume/CV: https://drive.google.com/uc?export=download&id=1GyjuKdbuBHgXjvptxVEpDUnPLofAKM6d
Website: https://www.amirnft.com/
Email: [amirhossain.limon@gmail.com](mailto:amirhossain.limon@gmail.com)
LinkedIn: https://www.linkedin.com/in/mdamirhossain-limon/

**Comment by pacifiquem** [comment]
Front-End Developer
Hi, I'm a 5 years-experience front-end developer (the last 3 years on web development), currently focused on Vue and Next.js. Very interested on the Jamstack architecture, and looking to gain full-stack skills.

Location: KIgali, Rwanda / (GMT+2)
Remote: Yes
Willing to relocate: No
Technologies: React, Next.js, Tailwind CSS, Typescript, CSS&HTML, Javascript (ES5, ES6), PostgreSQL, MongoDB, Mysql,Nest.js, Redux, Node.js, Express, SpringBoot, Hibernate .
Resume/CV: https://drive.google.com/file/d/1qv03qGTXMRfnprOxve6mBzsErQcxUNXo/view?usp=sharing
LinkedIn: https://www.linkedin.com/in/pacifique-murangwa-10394722b
Website/Portfolio: https://pacifiquem.engineer
Email: pacifiquemurangwa001 [at] gmail [dot] com

Thanks for the thread! Look forward to hearing from you soon :)

**Comment by zhao-create-xyz** [comment]
Location: Shenyang, Liaoning, China
Remote: Yes
Willing to relocate: No
Technologies: React, Next, Angular, Vue, Typescript, TailwindCSS, Node, Express, Nest, GraphQL, Django, Flask, Laravel, Fast API, ... 
Website: http://zhaoping.vercel.app
Email: zhaoping.dev@gmail.com
LinkedIn: https://linkedin.com/in/zhcharles

**Comment by MohamedBechirMejri** [comment]
Hello! I'm a full stack developer from Tunisia. I have been coding for more than two years and now I want to work with others on awesome projects so if you have any type of work for me (position at a company or startup, freelancing, a fun project with cool new ideas...) don't hesitate to contact me.

Location: Tunis, Tunisia
Remote: Yes
Willing to relocate: No
Phone: +216 55 209 052
Technologies: JavaScript, React, TypeScript, Next.JS, Firebase, MongoDB, Tailwind CSS, Node.js, Express.js... (check out my github for more)
Resume/CV: https://docs.google.com/document/d/1hqGKnaX8T88-WBTqPVfl14nxOmW2GPyE/
LinkedIn: https://www.linkedin.com/in/MohamedBechirMejri/
Website: https://MohamedBechirMejri.dev/
Email: [mohamedbechir.mejri.dev@gmail.com](mailto:mohamedbechir.mejri.dev@gmail.com)

**Reply by prooftv** [reply]
Hi, i have a cool project, that needs assembling all the pages from a purchased nextjs business directory template..

I also need to add a headless CMS: preferably Sanity..

Here's is the link to the template for previewing. 
By the way, i am a startup with no funds.

https://themes.getbootstrap.com/product/directory-directory-listing-react-theme/

**Comment by rawnly** [comment]
Hello! I'm a Software Engineer from Rome, Italy 🇮🇹! I have ~5 years of work experience on my shoulders and I'm currently looking for a remote position where I can express myself at 100%. I can learn new stuff very quickly and adapt to any condition :) 

Location: Rome, Italy
Remote: Yes
Willing to relocate: No problem
Technologies: Next, React, Typescript, Rust, Go, Node, Remix, Tailwind, Postgres, MongoDB, Mysql
Resume/CV: https://untitled.dev/resume 
Website: https://untitled.dev
Twitter: https://untitled.dev/twitter
Linkedin: https://untitled.dev/linkedin
Email: jobs@fedevitale.dev

**Comment by mastercodercat** [comment]
As a senior frontend developer specializing in React and Next.js with TypeScript, Shane(me) is now looking for new positions and growth opportunities. With over 9 years of experience in front-end development, I've worked as a frontend-heavy full stack developer, and participated in web/mobile projects with modern technologies and libraries, mainly with React, React Native, and Next.js.
I prefer to work on a growing team and startups with great teammates and prefer a fully-remote work environment.
I am available for full-time positions as well as part-time or contract positions in global teams.

Location: Canada
Remote: Yes
Willing to relocate: No
Technologies: **Next.js**, **React**, **React Native**, **Ionic**, **TypeScript**, **GraphQL**, **Cypress**, **RTL**, **Tailwind**, **Node.js**, **Express.js**, **Nest.js**, **Prisma**
Resume: [My Resume](https://drive.google.com/file/d/1fw3_udibFbQyhubnga_oqeejMLCgUA3n/view?usp=sharing)
Website: [Portfolio](https://shanelewis.vercel.app)
Email: shanelewix@gmail.com

**Comment by davidnoelopez** [comment]
Hi, I'm David Lopez a fullstack developer looking for new opportunities to grow. I been working in software for a while (10+ years) managing business processes, optimizing and automating everyday tasks. I've been working with React and Next.js for the past year and I'm looking to work and learn more with the tech stack.

Location: Monterrey, Mexico.
Remote: Yes
Willing to relocate: No
Technologies: Next.js, Node, React, PHP, Mongo.
Resume/CV: shorturl.at/DKX89
Website: https://github.com/davidnoelopez
Email: davidnoelopez@gmail.com

**Comment by maximum628** [comment]
Hi, I'm LiangZunJie and a full stack developer looking for new opportunities to grow. 
I am a professional web/app developer with 10+ years of experience in JavaScript and PHP frameworks.
I have already developed many web and app projects with React, Next.js, Angular, Vue.js, Node.js, SpringBoot and ,PHP, CI, Laravel, and Python.
I have strong experience in developing blockchain projects including smart contract development and front-end development using Solidity, Web3.js, etc.

Location: Georgia
Remote: Yes
Willing to relocate: No
Technologies: React, Next.js, Angular, Node, PHP, Mongo, MySQL.
Resume/CV: https://tinyurl.com/mve2ma6v
Website: https://github.com/maximum628
Email: [maximum95628@gmail.com](mailto:maximum95628@gmail.com)

**Comment by jinbest** [comment]
Hi, I'm Jin Zheng from China, and I'm looking for a job recently.

Location: China
Remote: Yes
Willing to relocate: No
Technologies: React, Next.js, JS/TS, Node, Express, MongoDB, Chrome-extension, Mobx, Vue, Quasar
Resume/CV: https://jin-zh.vercel.app/assests/resume.pdf
Website: https://jin-zh.vercel.app
Email: jin.bestvictoria718@gmail.com

Looking forward to hearing from you.
Thanks.

**Comment by js-matt** [comment]
Location: Chicago, Il
Remote: Yes
Willing to relocate: No
Technologies: TypeScript, React, Tailwind, Node.js, Serverless Functions, Vite, Vue, Next.js, Nuxt.js, Netlify, Vercel, AWS lambda, AWS S3, Styled Component, SCSS
Résumé/CV: https://drive.google.com/file/d/1ofRAolo2e9LOMVe5R--Q-30fl8POxn2B/view?usp=sharing
Website: https://github.com/js-matt
Email: [imatthewnguyen@gmail.com](mailto:imatthewnguyen@gmail.com)
LinkedIn: https://www.linkedin.com/in/matthew-nguyen-dev/

I am a team-oriented professional with over 6 years of experience in a high-volume distribution center. My natural love of problem solving and continual learning have aided me well in becoming a full-stack web developer, and I am eager to ﬁnd a team in which I can add value and with whom I can continue my growth as a software engineer

**Comment by dan-pugsley** [comment]
Hey! React/Next.js developer specialising in games and generally very complex/interactive UI with lots of movement and animation. I've worked on big projects like [Plague Inc: Evolved](https://store.steampowered.com/app/246620/Plague_Inc_Evolved/) (40,000+ reviews on Steam), smaller projects like [individual portfolios](https://leapoisson.com/) and I also have team lead experience. Check out my [portfolio](https://pugs.ly/). Message me for availability!

**Location:** Lisbon, Portugal
**Remote:** Yes
**Willing to relocate:** No
**Technologies:** React (JS/TS), Next.js, Node, SCSS, AWS... I learn very fast 😁
**Resume/CV:** https://s3.eu-west-2.amazonaws.com/pugs.ly/doc/resume.pdf
**Website:** https://pugs.ly/
**Email:** [dan@pugs.ly](mailto:dan@pugs.ly)

**Comment by kmoussai** [comment]
Hey, my name is khalid moussaid I'm a frontend developer with solid knowledge of the Javascript ecosystem especially ReactJs/NextJS with Typescript which I have been working with on a daily basis for the last two years. And I have more experience building maintainable and clean code with respect to best practices to deliver good quality apps.

Location: Morocco
Remote: Yes
Willing to relocate: Yes
Technologies: ReactJS, NextJS, Typescripts, CSS, SCSS, Node ... Eager to learn new things constantly
Resume/CV: https://drive.google.com/file/d/1-1EFPPwd1GWpcHFBWRF4mE77JQ3tgo5l/view?usp=sharing
Website: http://kmoussai.me/
Email: k.moussaid@outlook.com
My recent work: 
My recent work with NextJs with an Upwork client was to create a web app that generates credit cards with custom image, the client was Migros Bank in Switzerland.
you can check it on https://onboarding.cumulus.migrosbank.ch/ it's a production version with more than 30000+ credit cards with a custom image that was instant-issued starting from the launching date (1 July). and that's why you should choose me because I worked on an app that has been used by thousands of people and I have experience with production issues and working in a fast-paced environment.

**Comment by Keshavdulal** [comment]
Location: Kathmandu, Nepal (GMT +05:45)
Remote: Yes
Willing to relocate: Yes (Already applied to move to EST (UTC-5))
Technologies: React(6y), SASS(5y), RN(3+y), NextJS(2y), TypeScript(1y), GraphQL(2y), Tailwind(1y), Firebase(1y)
Resume/CV: https://bit.ly/keshav-dulal-resume-2022
Website: keshavdulal.com
Email: hello@keshavdulal.com

Recent Projects
1. Event Management System (Onboarding, Event Creation, Ticket Mgmt, Checkout, Dashboard)
Stack - React, Tailwind, Firebase, Cloud Functions, Stripe
https://sessami.co/

2. Deathbed (https://www.deathbed.app/)
- React Native (RN) app to statistically calculate life expectancy

Next JS Projects
1. https://innovations.vareximaging.com
2. https://www.bigdsignature.com

**Comment by devstar107** [comment]
Location: Sydney, Australia
Remote: Yes
Willing to relocate: I willing to relocate after 4 months
Technologies: React.js(7 years), Next.js(5), Typescript(5), WordPress(3), Vue.js(2), Node.js(5), MySql, MongoDB, PostgreSql
Resume/CV: https://drive.google.com/file/d/1OA1wXZQOZiLTAYKHhL1HJUWWtu2Bg52t/view?usp=sharing
Website: https://www.linkedin.com/in/harris-rui/
Email: [rui.harris.worK@outlook.com](mailto:rui.harris.worK@outlook.com)


Next.js + Wordpress Project

1. https://www.tuckertub.com.au
2. https://www.bikesociety.com.au
3. http://thegentsplace.com
4. http://multimedia-tree.com/

React Project

1. https://lighthouse.app
2. https://skillr.com
3. https://www.msbdocs.com/

**Comment by yhdesai** [comment]
Location: Mumbai, India
Remote: Yes
Willing to relocate: No
Technologies: Next.JS, Tailwind, Mongodb, Express, React, Node.js, Firebase, GCP
Resume/CV: https://linkedin.com/in/yhdesai
Website: https://yashdesai.dev
Email: project@yashdesai.dev

Most Recent Project: https://shipr.dev

**Comment by nemanjam** [comment]
Location: Serbia, GMT +1
Remote: Yes
Willing to relocate: Maybe
Technologies: React, Next.js, JavaScript/TypeScript, Node.js, TailwindCSS, Jest, Cypress, Prisma, PostrgreSQL, MongoDB, Docker, C#/.NET
Resume/CV: Available on request, please send me an email
Website: https://github.com/nemanjam
LinkedIn: https://www.linkedin.com/in/nemanja-mitic
Email: nemanja.mitic.elfak [at] hotmail.com


I am a full stack developer with a Master's degree in Computer Science and almost 4 years of professional experience with JavaScript and C#/.NET. In the last few years my main focus has been the React/Next.js ecosystem, followed by Node.js. I am a JavaScript professional and a Python beginner interested to get more exposure to it. I have also worked with ASP.NET in the past which is not my primary focus at the moment. I have a basic understanding of DevOps tools.

A quick and easy way to estimate my current skill level, the quality of my code, and my preferred tech stack (Next.js, TypeScript, React Query, Prisma, Postgres, TailwindCSS, Docker, Github Actions) is to check out my latest project: https://github.com/nemanjam/nextjs-prisma-boilerplate .

I am looking for a full-time remote position. We can discuss relocation after some time spent in the company. You can check out my other projects at https://github.com/nemanjam . I have detailed resume available, you can tell me about your company and project by email: nemanja.mitic.elfak [at] hotmail.com or by LinkedIn: https://www.linkedin.com/in/nemanja-mitic .

**Comment by codexandr** [comment]
Location: Canada
Remote: Yes
Willing to relocate: Only Remote
Technologies: React, Next.js, JavaScript/TypeScript TDD, GraphQL, RDBMS, NoSQL, Docker,
Resume/CV: https://drive.google.com/file/d/1UKnSIr1Od678v366bSZ-yqFBsTI1EMFR/view?usp=share_link
Website: https://github.com/hotbrainy
LinkedIn: https://www.linkedin.com/in/hotbrainy
Email: brainhg95@gmail.com

**Comment by bouyagas** [comment]
On Tue, Dec 6, 2022 at 13:26 Hassan El Mghari ***@***.***>
wrote:

> What is this?
>
> This discussion is for Next.js developers looking for work.
> Candidates
>
> Share your information if you are looking for work. Please use this format:
>
> Location:
> Remote:
> Willing to relocate:
> Technologies:
> Resume/CV:
> Website:
> Email:
>
> Good luck with interviewing!
> ------------------------------
>
> If you got hired through this job board, please reach out to me (
> ***@***.***) so I can send you some swag :)
>
> If you're a company looking to post a role, use #42311
> <https://github.com/vercel/next.js/discussions/42311>.
>
> *If you're looking for last month's discussion, see #42310
> <https://github.com/vercel/next.js/discussions/42310>. We've had another
> success story <https://twitter.com/Nutlope/status/1475929984174895105> of a
> developer being hired!*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vercel/next.js/discussions/43786>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABOI4PZCVNJNT2QV5T4RVADWL6ALTANCNFSM6AAAAAASV236JU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Sent from Gmail Mobile

**Comment by frontsunriver** [comment]
Location: Azerbaizan
 Remote: Yes
 Willing to relocate: Only Remote
 Technologies: ReactJS(NextJS), AngularJS, PHP
 Resume/CV: www.linkedin.com/in/murad-aliasgarov-3192a8256
 Website: 
 Email: muradaliasgarov01@gmail.com

**Comment by JonathanGrossman** [comment]
Hi all. I am a fullstack developer who specializes in building complex applications comprised of multiple microservices and usually some sort of machine learning. I currently am building a video chatbot that leverages machine learning to answer questions with videos. I also train junior developers to help them build a developer portfolio. I'm looking for paid freelance work or for devs who want to join my community and make useful technology.

Location: Israel
Remote: Yes
Willing to relocate: No
Technologies: Next.js, Node/Express, React, JavaScript, Python, Flask, Google Cloud Platform, Google Colab, TensorflowJS, MongoDB, SQL, WordPress, RaspberryPi
Website: https://thacash.com
LinkedIn: https://www.linkedin.com/in/jonathan-grossman-06867a4/
Email: [jon@thacash.com](mailto:jon@thacash.com)

**Comment by codewithPoppy** [comment]
Location: Pittsburgh, PA
Remote: Yes
Willing to relocate: Yes
Technologies: Next.js 13.0, React 18.0, Blitz.js, TypeScript, SEO, SSR, CMS(Contentful, Storyblok), Web vitals, GraphQL
LinkedIn: https://www.linkedin.com/in/damian-lee-22791b248/
Email: poppy.dev0920@gmail.com

**Comment by kunxl-gg** [comment]
Location: Mumbai, India
Remote: Yes
Willing to relocate: No
Technologies: React.js, Django, typescript and flutter 
Email: tiwari.25@iitj.ac.in
Linkedin:  www.linkedin.com/in/kunal-tiwari-3b0bb71b6

**Comment by raynirola** [comment]
Location: Sikkim, IN
Remote: Yes
Willing to relocate: No
Technologies: React, Nextjs, TypeScript, Node, AWS. 

Email: ray@nirola.in
Website: https://nirola.in

**Comment by kelvinsanchez15** [comment]
Location: Spring, TX, United States
Remote: Yes
Willing to relocate: Only Remote
Technologies: React, Next.js, JavaScript/TypeScript, Redux, Node, Material UI, TDD
Resume/CV: https://www.kelvinsanchez15.com/resume/resume-en.pdf
Website: https://github.com/kelvinsanchez15
LinkedIn: https://www.linkedin.com/in/kelvin-sanchez
Email:  kelvinsanchez15@gmail.com

**Comment by Mugisha-isaac** [comment]
Location: Rwanda
Remote: Yes
Willing to relocate: No
Technologies: Nodejs, Reactjs, Docker, Nextjs, Express js, Kafkajs, springboot, Nestjs, Remix, Typescript, css & tailwind
Resume/CV: 
[MUGISHA ISAAC.pdf](https://github.com/vercel/next.js/files/10231733/MUGISHA.ISAAC.pdf)

Website:  https://github.com/Mugisha-isaac
Email:  mugishaisaac0508@gmail.com

**Comment by jagoral** [comment]
Location: Poland
Remote: Yes
Willing to relocate: Maybe
Technologies: Next.js, React.js, Vue.js, Node.js, Nest.js, Prisma, Testing Library, Chakra UI, Mantine, Tailwind, Cypress, Contentful, 
Resume/CV: [TomaszGóral_CV.pdf](https://github.com/vercel/next.js/files/10247378/TomaszGoral_CV.pdf)
Website: - (I can share some repos during interview process)
Email: tomasz.goral@outlook.com

**Comment by abayomi1196** [comment]
Hello! I'm Ayo.

I'm a Front-end developer with working experience in NextJS, Svelte, JavaScript/TypeScript, React, CSS/SCSS, Tailwind CSS, and Git, among others.

I am always looking to learn new skills and improve on existing knowledge, hoping to contribute to making the world a better place.

Remote: Yes
Willing to relocate: Yes
Location: Nigeria (GMT+1)
Email: [ayodele@duck.com](mailto:ayodele@duck.com)
Website: https://portfolio-ayodele.vercel.app/
Resume: [Resume](https://drive.google.com/file/d/1_9YR482swkUOd2dHSXZJx5B1zB5F2E2I/view?usp=sharing)

**Comment by saiful-70** [comment]
I'm a passionate and enthuastic programmer. My core skill is based on JavaScript. I'm learning and implementing javascript and it's related technology for 3+ years.
I am available for any kind of job opportunity that suits my interests.

Location: Bangladesh
Remote: Yes
Willing to relocate: No
Technologies: React, NextJs, Material UI, TailwindCSS, SCSS, Redux, Typescript, Git
Resume/CV: https://www.saifuls.xyz/resume/resume-saiful.pdf
Website: https://www.saifuls.xyz/
Email: saiful70.me@gmail.com

**Comment by adebiyial** [comment]
Location: Lagos, Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next.js, Remix, Styled Components, TypeScript, Tailwind CSS, etc.
Resume/CV: https://www.linkedin.com/in/adebiyiadedotun
Website: https://orangeman.dev
Email: adebiyiadedotun9@gmail.com

**Comment by Chimise** [comment]
Location: Lagos, Nigeria,
Remote: Yes,
Willing to relocate: yes,
Technologies: React, Next.js, Node.js, Tailwind CSS, MongoDB, Postgresql, JavaScript/ Typescript, Strapi CMS
Resume: https://chimisepro.vercel.app/chisom_frontend.pdf
Website: https://chimisepro.vercel.app/
Email: chimisepro@gmail.com
Happy to Connect

**Comment by GeauxWeisbeck4** [comment]
Location: Raleigh, NC
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next.js, Node.js, Deno, SASS/SCSS, Tailwind CSS, JavaScript, PostgreSQL, MongoDB, Storyblok CMS, Strapi CMS, Typescript, Python 
Resume/CV: https://geauxweisbeck4.dev/resume
Website: https://geauxweisbeck4.dev
Email: andrew.weisbeck@gmail.com

Hoping to connect with someone!

**Comment by BadarSaghir** [comment]
Front-End/Backend Developer
Hi, I'm a Sardar Badar, working as a freelancer on fiverr for 2 years. I have not only have expertise in next.js and reactjs but also worked  backed mainly  with nestjs and express. Here are the details

Location:Pakistan/ (GMT +5)
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next.js, electron, svelte, Tailwind CSS, Typescript, Javascript, mongodb, firebase, PostgreSQL, Zustand, Redux, Node.js, Express, selenium, python, linux
Resume/CV: https://github.com/BadarSaghir/BadarSaghir/blob/d23daa4ce14160a3c6543e4484aa384b7768e273/resume.pdf
LinkedIn: https://www.linkedin.com/in/sardar-badar-saghir-sardar-saghir-ahmad/
Website/Portfolio: https://www.github.com/BadarSaghir
Email: dataentrybadar@gmail.com
Look forward to hearing from you soon :)
